bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/CameraAddress.java

```diff
@@ -25,46 +23,39 @@
 @NonNullByDefault
 public class CameraAddress {
 
-    private final String vpnURL;
-    private final String localURL;
+    private final String vpnURL; // https://prodvpn-eu-2.netatmo.net/restricted/10.255.39.58/f48daf5f0e366bc8f9c397fbc73220e7/MTYyNDQ1NjgwMDrf1ChChrvaKZolVvWtHuuxT8_EiA,,
+    private final boolean local;
+    private @Nullable String localURL; // http://192.168.0.65/f48daf5f0e366bc8f9c397fbc73220e7
```

Comment:
I am not sure that is expected to have these comments, in particular the second with a hardcoded IP address

Reply:
Removed